color.go

```diff
@@ -0,0 +1,54 @@
+package termui
+
+import "github.com/fatih/color"
+
+// These mirror their equivalent functions in ui.go
+// Unfortunately since Print is already variadic,
+// we can't have an elegant "...color.Attribute" argument and have to use a slice
```

Comment:
The first argument (to `Sprintf`) is a string, so no worries there.

(This is _not_ an argument for actually doing it, just… going down the rabbit hole.)